### PR TITLE
DTOs defined

### PR DESCRIPTION
**Pull request by dawidkeshop**

### CODE REVIEW

File: src/CaptainHook.Api/Dtos/AuthenticationDto.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
```
Comment (chomick) [inline src/CaptainHook.Api/Dtos/AuthenticationDto.cs:1]: I would suggest to have the DTOs in a separate 'contract' project but can be updated in subsequent PR, if agreed to move

File: src/CaptainHook.Api/Dtos/AuthenticationDto.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace CaptainHook.Api.Dtos
+{
+    public class AuthenticationDto
```
Comment (chomick) [inline src/CaptainHook.Api/Dtos/AuthenticationDto.cs:5]: I was thinking - what if we simply exposed the existing KV models (temporarily) so we can see all the data properly?
The problem with original approach is that we need to map from KV to new DTOs and we will lose lots of info which possibly is very useful to have, even if format is temporary and there is no clear separation between API contract and internal models for now.

Just a thought.
The ticket did allow for bits to be skipped but I wonder if it makes sense to do it that way, considering where we are now we the knowledge/work/plans.

Alternatively, maybe we can expose this GET +additional GET (or section within subscriber) where KV settings are out there in raw (minus secrets)?

The benefit would be that we can see the actual settings that are really used by CH.

File: src/CaptainHook.Api/Dtos/AuthenticationDto.cs (hunk 1 above)
Comment (dawidkeshop) [reply src/CaptainHook.Api/Dtos/AuthenticationDto.cs]: OK, makes sense.

File: src/CaptainHook.Api/Dtos/AuthenticationDto.cs (hunk 2 above)
Comment (dawidkeshop) [reply src/CaptainHook.Api/Dtos/AuthenticationDto.cs]: We can do anything but what are the actual requirements there? If we need another secret API endpoints for some test purposes, we should define the story in a different way.

**Pull request merged**